torchqdynamics/mesolve/solver_options.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+
+from ..solver import FixedStep
+
+
+@dataclass
+class Rouchon1(FixedStep):
+    pass
+
+
+@dataclass
+class Rouchon1_5(FixedStep):
+    pass
+
+
+@dataclass
+class Rouchon2(FixedStep):
+    pass
```

Comment:
As I was suggesting, I think it would be nice to have a default `Rouchon` solver that redirects to either `Rouchon1` or `Rouchon2` by default. This way users can simply call `Rouchon` without having to bother about the backend order of the solver.